<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="AGUMMADILLI06" failures="0" tests="1" name="Default test" time="0.051" errors="0" timestamp="2024-07-29T13:45:51 IST">
  <testcase classname="CoreJava.clsExceptionHandling" name="ArrayIndexException" time="0.007"/>
  <testcase classname="CoreJava.clsExceptionHandling" name="LearnException2" time="0.0">
    <ignored/>
  </testcase> <!-- LearnException2 -->
  <testcase classname="CoreJava.clsExceptionHandling" name="LearnException1" time="0.0">
    <ignored/>
  </testcase> <!-- LearnException1 -->
  <testcase classname="CoreJava.clsExceptionHandling" name="LearnExceptionHandling" time="0.0">
    <ignored/>
  </testcase> <!-- LearnExceptionHandling -->
</testsuite> <!-- Default test -->
